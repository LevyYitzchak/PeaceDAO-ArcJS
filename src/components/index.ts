export * from "./DAO";
export * from "./DAOs";
export * from "./Member";
export * from "./Members";
// export * from "./proposals";
export * from "./Reputation";
export * from "./Reputations";
export * from "./Token";
export * from "./Tokens";
export * from "./Reward";
export * from "./Rewards";
export * from "./Vote";
export * from "./Votes";
export * from "./Stake";
export * from "./Stakes";
export * from "./plugins";
