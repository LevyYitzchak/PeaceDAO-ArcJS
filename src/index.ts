export * from "./runtime";
export * from "./protocol";
export * from "./components";
