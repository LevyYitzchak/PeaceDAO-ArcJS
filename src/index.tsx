// web3
// graphql server
// contract addresses / ABIs
// logging ComponentLogs.enabled = true
