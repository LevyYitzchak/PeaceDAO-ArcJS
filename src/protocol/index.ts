export * from "./Arc";
export * from "./ArcConfig";
export * from "./settings";
