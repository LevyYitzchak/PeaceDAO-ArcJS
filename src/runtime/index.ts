export * from "./Component";
export * from "./ComponentList";
export * from "./Protocol";
export * from "./logging/index";
export * from "./Loader";
