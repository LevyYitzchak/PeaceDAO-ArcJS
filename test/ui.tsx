import React = require("react");
import { create } from "react-test-renderer";
import { SampleWidget } from "../src";

test("Say my name, say my name...", () => {
  const tree = create(
    <SampleWidget name="Visitor" />
  ).toJSON();
  expect(tree).toMatchSnapshot();
})
