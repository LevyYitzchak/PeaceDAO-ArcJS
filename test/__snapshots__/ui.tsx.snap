// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Say my name, say my name... 1`] = `
<div>
  <h1>
    Hello 
    Visitor
  </h1>
</div>
`;
